### PR TITLE
dry run, cve file empty. sharp package updated.

### PR DESCRIPTION
**Pull request by farrukhishaq**
Testing and experimenting CI/CD, CVE affect towards ITHC 2024.

### CODE REVIEW

File: .auditrc.json [hunk 1]
```diff
@@ -1,8 +1,9 @@
 {
```
Comment (farrukhishaq) [inline .auditrc.json:1]: when running locally cmd `npm run audit:test` will use this file to accept vul from the allowList.

File: .drone-1.0.yml [hunk 2]
```diff
@@ -42,12 +41,15 @@ steps:
...
+    image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/trivy/client:latest
+    resources:
+      limits:
+        cpu: 1000
+        memory: 1024Mi
     environment:
       IMAGE_NAME: asl-attachments
-      WHITELIST: CVE-2022-3734, CVE-2018-11218, CVE-2022-0543, CVE-2018-11219, CVE-2021-29940, CVE-2022-32511
-      WHITELIST_FILE: cve-exceptions.txt
+      ALLOW_CVE_LIST_FILE: cve-exceptions.txt
```
Comment (farrukhishaq) [inline .drone-1.0.yml:52]: `cve-exceptions.txt`  allowList of CVE VUL for the docker image.

File: .gitignore [hunk 3]
```diff
@@ -102,3 +102,9 @@ dist
 
 # TernJS port file
 .tern-port
+
+# InteliJ's
+.idea/*
```
Comment (jeff-horton-ho-sas) [inline .gitignore:107]: Also exclude root intellij `.iml` file from root

```suggestion
.idea/*
/*.iml
```

File: asl-attachments.iml
Comment (jeff-horton-ho-sas) [inline asl-attachments.iml:1]: This is IDE config and should be in .gitignore

**Review by jeff-horton-ho-sas — COMMENTED** [review]
Changes look good. An Intellij config file has snuck into the commit, but otherwise happy to approve.

**Review by jeff-horton-ho-sas — APPROVED** [review]

**Pull request merged**